### PR TITLE
Initial rewrite of windows install

### LINKED ISSUE #40280: Consider ways to make this page look less hard
### Type of issue

General feedback on page

### Description

This page seems very intimidating. I don't know if there are limitations that keep us from having separate pages here, which might help a bit. Mostly this content, especially install with VS jumps straight to information an expert needs (which TFM) and not what a beginner needs "You have what you need and this is how you adjust installed workloads if you need that"

Since most folks use VS, could we move that to the top and start with friendly language like "If you're using Visual Studio, hopefully you do not need anything on this page". OK, not that ;-)

Or put DevKit installation at the top with a nice simple walk-through. 

Installation seems a place to focus on a welcome, feel good, hand holding vibe. 

This page makes it look like it is hard to install .NET, and it came up high in SEO.

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net80#install-with-visual-studio

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/install/windows.md

### Document Version Independent Id

d11d0d14-8a03-50e4-5c00-9a4417ad4d0a

### Article author

@adegeo

### Metadata

* ID: 1eeab451-820f-de42-983e-4d0d5764f6ab 
* Service: **dotnet-fundamentals**
* Sub-service: **install**

---
[Associated WorkItem - 239918](https://dev.azure.com/msft-skilling/Content/_workitems/edit/239918)

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added the **Choose how to install .NET** section to guide the audiences.
- Added the **Which runtime should you install** section at the start to describe the runtimes. This was previously repeated a  lot in the article under individual sections.
- Rewrite article to follow `install-set-up-deploy` linter template.
- Rewrite all the **Install...** sections to have an introduction then steps if appropriate.
- Added steps to some **Install...** sections in addition to linking to the official docs that show that method.
- **Windows Installer** section renamed to **.NET Installer** and language changed to be more new-user friendly.
- Merged some of the support tables.
- Reduced the size of the **Runtime** and **SDK** sections under different **Install...** sections.
- Added an additional troubleshooting item.
- Reduced the Windows 7 content.
- Removed the tabs for versions. I was able to merge the content into a single section.
- Trim up WinGet section to be concise.

## Todo

- [x] Edit the Arm64 support table. It no longer makes sense here and in macos. .NET 5 was the last version to have an issue and that's been long retired.
- [ ] Reconsider the whole design of the page. I'm not sure that we want to break up all the install methods into individual articles. That becomes a maintenance nightmare as they'll require deeper instructions. The docs for those products already include detailed instructions on how to install things.
- [ ] Another idea would be to move the ARM info into its own article.

@KathleenDollard 

Fixes #40280

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/macos.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/core/install/macos.md) | [Install .NET on macOS](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-40999) |
| [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/core/install/windows.md) | [Install .NET on Windows](https://review.learn.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-40999) |
| [docs/core/rid-catalog.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/core/rid-catalog.md) | [.NET Runtime Identifier (RID) catalog](https://review.learn.microsoft.com/en-us/dotnet/core/rid-catalog?branch=pr-en-us-40999) |
| [docs/core/runtime-discovery/troubleshoot-app-launch.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/core/runtime-discovery/troubleshoot-app-launch.md) | [Troubleshoot app launch failures](https://review.learn.microsoft.com/en-us/dotnet/core/runtime-discovery/troubleshoot-app-launch?branch=pr-en-us-40999) |
| [docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/core/tools/dotnet-install-script.md) | [docs/core/tools/dotnet-install-script](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script?branch=pr-en-us-40999) |
| [docs/devops/dotnet-cli-and-continuous-integration.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/devops/dotnet-cli-and-continuous-integration.md) | [Use the .NET SDK in continuous integration (CI) environments](https://review.learn.microsoft.com/en-us/dotnet/devops/dotnet-cli-and-continuous-integration?branch=pr-en-us-40999) |
| [docs/standard/glossary.md](https://github.com/dotnet/docs/blob/6fa6106ae2ecc8701263bb62c3d55bc3c6ffce6d/docs/standard/glossary.md) | [.NET glossary](https://review.learn.microsoft.com/en-us/dotnet/standard/glossary?branch=pr-en-us-40999) |

### CODE REVIEW

File: docs/core/install/windows.md [hunk 1]
```diff
@@ -166,211 +181,192 @@ If you already have Visual Studio installed, you can check your version with the
...
-> [Download Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019).
-
-For more information, see [.NET SDK, MSBuild, and Visual Studio versioning](../porting/versioning-sdk-msbuild-vs.md).
+For more information about, see [.NET SDK, MSBuild, and Visual Studio versioning](../porting/versioning-sdk-msbuild-vs.md).
 
-### Select a workload
+## Install with Visual Studio Code
 
-When installing or modifying Visual Studio, select one or more of the following workloads, depending on the kind of application you're building:
+Visual Studio Code uses the versions of .NET already installed on your system. Remember, Visual Studio installs its own copy of .NET, so you must install .NET using either [.NET Installer](#net-installer) or [Windows Package Manager](#install-with-windows-package-manager-winget).
```
Comment (CamSoper) [inline docs/core/install/windows.md:188]:
```suggestion
Visual Studio Code uses the versions of .NET already installed on your system. Install .NET using either [.NET Installer](#net-installer) or [Windows Package Manager](#install-with-windows-package-manager-winget). Visual Studio installs its own copy of .NET that can't be used by Visual Studio Code.
```

File: docs/core/install/windows.md [hunk 2]
```diff
@@ -166,211 +181,192 @@ If you already have Visual Studio installed, you can check your version with the
...
-Environment variables that add .NET to system path, such as the `PATH` variable, may need to be changed if you have both the x64 and Arm64 versions of the .NET SDK installed. Additionally, some tools rely on the `DOTNET_ROOT` environment variable, which would also need to be updated to point to the appropriate .NET SDK installation folder.
+    ```cmd
+    winget install Microsoft.DotNet.SDK.8
+    ```
 
-## Dependencies
+### Install the runtime
 
-# [.NET 8](#tab/net80)
+There are different runtimes you can install. Refer to the [Which runtime should you install](#which-runtime-should-you-install) section to understand what's included with each runtime.
```
Comment (CamSoper) [inline docs/core/install/windows.md:274]:
```suggestion
There are different runtimes you can install. Refer to the [Choose the correct runtime](#choose-the-correct-runtime) section to understand what's included with each runtime.
```

**Review by CamSoper — CHANGES_REQUESTED** [review]
Mostly LGTM, made some revisions for wording and sentence structure

File: docs/core/install/windows.md [hunk 3]
```diff
@@ -14,139 +16,154 @@ ms.date: 05/14/2024
...
+## Supported versions
 
-```cmd
-winget install Microsoft.DotNet.DesktopRuntime.Preview
-```
+The following table is a list of currently supported .NET releases and the versions of Windows they're supported on. These versions remain supported until either the version of [.NET reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Windows reaches end-of-life](https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet).
 
-## Install alongside Visual Studio Code
+> [!TIP]
+> As a reminder, this table applies to modern .NET (as opposed to .NET Framework). .NET Framework downloads can be found [here](https://dotnet.microsoft.com/download/dotnet-framework).
```
Comment (adegeo) [inline docs/core/install/windows.md:76]:
```suggestion
> As a reminder, this table applies to modern .NET (as opposed to .NET Framework). To install .NET Framework, see the [.NET Framework Installation guide](../../framework/install/index.md).
```

File: docs/core/install/windows.md [hunk 4]
```diff
@@ -35,13 +35,12 @@ There are three different runtimes for Windows, which enable different types of
 | **ASP.NET Core Runtime** | No                            | No                            | Yes                           |
 | **.NET SDK**             | Yes                           | Yes                           | Yes                           |
 
-To ensure that you can run all .NET apps on Windows, install both the ASP.NET Core Runtime and the .NET Desktop Runtime.
```
Comment (CamSoper) [inline docs/core/install/windows.md]: Blank line failing the lint

**Review by CamSoper — APPROVED** [review]

**Pull request merged**